integration/utils_test.go

```diff
@@ -341,3 +341,17 @@ func GenerateRequests() ([]api.GenerateRequest, [][]string) {
 			[]string{"nitrogen", "oxygen", "carbon", "dioxide"},
 		}
 }
+
+func EmbedTestHelper(ctx context.Context, t *testing.T, req api.EmbedRequest) (*api.EmbedResponse, error) {
```

Comment:
is this used outside of `embed_test.go`? If not, let's put it in there as `embedTestHelper`